src/hex_tarball.erl

```diff
@@ -13,6 +13,33 @@
     {<<"Makefile">>, <<"make">>},
     {<<"Makefile.win">>, <<"make">>}
 ]).
+
+-define(DEPRECATED_META_FIELDS, [
+            <<"maintainers">>,
+            <<"contributors">>
+]).
+
+-define(META_FIELDS, [
+                       <<"name">>,
+                       <<"version">>,
+                       <<"files">>,
+                       <<"build_tool">>,
+                       <<"build_tools">>
```

Comment:
the full list of fields is here: https://github.com/hexpm/specifications/blob/master/package_metadata.md.

There's no `build_tool` field - have you encountered it in the wild?

Reply:
In the tests yes ;)  I think I did a push that removes this field and updates the tests.

Reply:
haha, oops!